_episodes/08-ci.md

```diff
@@ -1,33 +1,37 @@
 ---
-layout: page
-title: Testing
-subtitle: Continuous Integration
-minutes: 10
+title: Continuous Integration
+teaching: 10
+exercises: 0
+questions:
+- "How can I make sure my changes are compatable with the larger code base?"
```

Comment:
No. This is about automating tests. "How can I automate running the tests on more platforms than my own?"